runtime/lua/vim/treesitter/dev.lua

```diff
@@ -3,16 +3,21 @@ local api = vim.api
 ---@class TSDevModule
 local M = {}
 
+---@private
 ---@class TSTreeView
 ---@field ns integer API namespace
----@field opts table Options table with the following keys:
----                  - anon (boolean): If true, display anonymous nodes
----                  - lang (boolean): If true, display the language alongside each node
----                  - indent (number): Number of spaces to indent nested lines. Default is 2.
+---@field opts TSTreeViewOpts
 ---@field nodes TSP.Node[]
 ---@field named TSP.Node[]
 local TSTreeView = {}
 
+---@private
+---@class TSTreeViewOpts
+---@field anon boolean If true, display anonymous nodes. Defaults to false.
+---@field lang boolean If true, display the language alongside each node. Defaults to false.
+---@field range boolean If true, display 0-based treesitter range instead of 1-based line/col. Defaults to false.
+---@field indent number Number of spaces to indent nested lines. Default value is 2.
```

Comment:
If these are no longer nullable, then there's no need to indicate the default value.